src/pca9685.rs

```diff
@@ -86,16 +86,20 @@ impl Pca9685DeviceBuilder {
     /// Builds the `Pca9685Device`.
     pub fn build(self) -> Result<Pca9685Device, Box<dyn Error>> {
         let device = I2cdev::new(self.i2c_bus)?;
-        let pwm = Pca9685::new(device, self.address).expect("Failed to open PWM controller");
+        let mut pwm = Pca9685::new(device, self.address).expect("Failed to open PWM controller");
 
         let oe_pin = {
             let pin = Pin::new(self.oe_pin_number);
             pin.export()?;
             sleep(Duration::from_millis(30));
-            pin.set_direction(Direction::Low)?;
+            pin.set_direction(Direction::High)?;
             pin
         };
 
+        pwm.reset_internal_driver_state();
```

Comment:
Yes, the PWM was set up to be enabled/activated from the OE pin, which uses inverted logic.
Therefore, the PWM's enable register remains enabled.